README.md

```diff
@@ -328,10 +328,13 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
 - :books: [Best roadmap](https://vitto.cc/web3-and-solidity-smart-contracts-development-roadmap/#d12f)
 - :video_camera: [Web3 Roadmap](https://www.youtube.com/watch?v=Lz3m4BI1EfY&t=5s)
   
-  ---
   
-  DevOps 
+---
+  
+ ## DevOps 
```

Comment:
add DevOps resources in alphabetical order also make sure to add 
```
[DevOps](#devops) 
```
in index as well in alphabetical order too

Reply:
ok